### ISSUE TITLE
Can't open a console with rails master

### ISSUE THREAD

**Issue by gregmolnar** [basecamp/console1984#29]
@jorgemanrubia I wanted to try to bypass the protection after the latest improvements, but I can't even open a console due to a weird ActiveRecord error:

```
Unnamed, why are you using this console today?
testing...
Traceback (most recent call last):
	32: from bin/rails:5:in `<main>'
	31: from bin/rails:5:in `require'
	30: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/commands.rb:18:in `<top (required)>'
	29: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/command.rb:48:in `invoke'
	28: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/command/base.rb:87:in `perform'
	27: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
	26: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
	25: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
	24: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/commands/console/console_command.rb:102:in `perform'
	23: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/commands/console/console_command.rb:19:in `start'
	22: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/commands/console/console_command.rb:19:in `new'
	21: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/commands/console/console_command.rb:35:in `initialize'
	20: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/engine.rb:449:in `load_console'
	19: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/application.rb:522:in `run_console_blocks'
	18: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/engine/railties.rb:15:in `each'
	17: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/engine/railties.rb:15:in `each'
	16: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/application.rb:522:in `block in run_console_blocks'
	15: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/railtie.rb:250:in `run_console_blocks'
	14: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/railtie.rb:275:in `each_registered_block'
	13: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/railtie.rb:275:in `each'
	12: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/railties/lib/rails/railtie.rb:250:in `block in run_console_blocks'
	11: from /Users/gregmolnar/Git/console1984/lib/console1984/engine.rb:24:in `block in <class:Engine>'
	10: from /Users/gregmolnar/Git/console1984/lib/console1984/supervisor.rb:25:in `start'
	 9: from /Users/gregmolnar/Git/console1984/lib/console1984/supervisor.rb:51:in `start_session'
	 8: from /Users/gregmolnar/Git/console1984/lib/console1984/sessions_logger/database.rb:8:in `start_session'
	 7: from /Users/gregmolnar/Git/console1984/lib/console1984/sessions_logger/database.rb:53:in `silence_logging'
	 6: from /Users/gregmolnar/Git/console1984/lib/console1984/sessions_logger/database.rb:9:in `block in start_session'
	 5: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/activerecord/lib/active_record/querying.rb:22:in `find_or_create_by!'
	 4: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/activerecord/lib/active_record/relation.rb:182:in `find_or_create_by!'
	 3: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/activerecord/lib/active_record/relation/finder_methods.rb:81:in `find_by'
	 2: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/activerecord/lib/active_record/relation/query_methods.rb:715:in `where'
	 1: from /Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/activerecord/lib/active_record/relation/query_methods.rb:720:in `where!'
/Users/gregmolnar/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/rails-a692e63bf400/activerecord/lib/active_record/relation/query_methods.rb:124:in `where_clause': uninitialized constant ActiveRecord::QueryMethods::Relation (NameError)
Did you mean?  ActiveRecord::Relation
```

If I remove the `console1984` from my `Gemfile`, `ActiveRecord` works as expected. Do you have any idea what causes this?

**Comment by jorgemanrubia** [comment]
Hey @gregmolnar,

This is weird. We are using current version in production without problem but I can reproduce in the gem "dummy" test app. I still need to figure out root cause, but I just pushed a patch that should fix the issue. 

Could you try the gem directly from git and confirm it now works for you?

```ruby
gem "console1984", git: "https://github.com/basecamp/console1984.git"
```

Thanks!

**Comment by gregmolnar** [comment]
It works now, thanks for the quick response!

**Comment by jorgemanrubia** [comment]
Great thanks @gregmolnar. I'll keep this open to remember I need to release a new version with this for RubyGems. Will do today!

**Comment by jorgemanrubia** [comment]
Released 0.1.13 with the patch for this one.
